java/core/src/java/org/apache/orc/AggregateStatistics.java

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
This should only have /* and not /**.

Reply:
We don't need this file.

Reply:
right, this was the artifact of refactoring, will remove it.